openprescribing/templates/faq.html

```diff
@@ -72,9 +74,22 @@ <h3 id="cite">How do I cite OpenPrescribing?</h3>
 <h3 id="timeperiod">What time period does the dataset cover?</h3>
 <p>We have data from the past 5 years on OpenPrescribing.net, and from 1998 in the <a href="https://openprescribing.net/long_term_trends/">Long Term Trends section</a>.</p>
 
+<h3 id="olddata">How can I access data older than available on your website?</h3>
+<p>Given our limited resources, we do not routinely provide data not accessible through the website. However, you may be able to access this data yourself. The NHS BSA openly publishes the <a href="https://opendata.nhsbsa.net/dataset/english-prescribing-data-epd">English prescribing dataset</a> dating back to January 2014. If data prior to 2014 is required, the <a href="https://www.nhsbsa.nhs.uk/statistical-collections/prescription-cost-analysis-england">Prescription cost analysis dataset</a> is accessible from 2004 onward.</p>
```

Comment:
should we add something like "although the PCA data is not as detailed as the English Prescribing Dataset?